.rpm-lockfiles/check-repo-access.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Quick check of repository access for Submariner RPM dependencies
+#
+# Tests if repos are accessible (subscription entitlements).
+# For full package verification, use: verify-packages.sh
+#
+# Requires: curl, entitlement certs in /etc/pki/entitlement/
+
+set -euo pipefail
+
+# Colors
+R='\033[31m' G='\033[32m' B='\033[1m' N='\033[0m'
+
+# Find entitlement cert
+CERT=$(find /etc/pki/entitlement -name "*.pem" ! -name "*-key.pem" 2>/dev/null | head -1)
+KEY=${CERT%.pem}-key.pem
+[[ -f "$CERT" ]] || { echo "No entitlement certs. Run: sudo subscription-manager register"; exit 1; }
+
+# Check repo access (returns 0=accessible, 1=blocked)
+check() {
+    local code
+    code=$(curl -s -o /dev/null -w "%{http_code}" -k --cert "$CERT" --key "$KEY" "$1" 2>/dev/null) || code=000
+    [[ $code == 200 ]]
+}
+
+check_public() {
+    local code
+    code=$(curl -s -o /dev/null -w "%{http_code}" "$1" 2>/dev/null) || code=000
+    [[ $code == 200 ]]
```

Comment:
Since the HTTP response code is a number, this should use numerical comparison; instead of using a placeholder code on failure, rely on the status code from the variable assignment (which is the status code from the `curl` invocation):

```suggestion
    code=$(curl -s -o /dev/null -w "%{http_code}" "$1" 2>/dev/null) && [[ $code -eq 200 ]]
```